### PR TITLE
Feature/Update Repo

### PR DESCRIPTION
**Pull request by juanpflores**
Hey @bernardopires,
Great Repo! I added the next changes to the repo in order to allow it to work with the new Auth system for the Twitch API:

- Added the API Dictionary in the settings file.
- Send the API key via headers when you request the top 100 streamers. 
- Added necessary documentation on the README file.

It's working on my computer. Let me know if you need me to make any changes.

### CODE REVIEW

File: settings.py.example [hunk 1]
```diff
@@ -1,15 +1,19 @@
 IRC = {
-    'SERVER': 'irc.twitch.tv',
-    'NICK': 'twitch_username',
-    'PASSWORD': 'your_oauth_password',  # tip: http://twitchapps.com/tmi/
+    'SERVER': 'irc.chat.twitch.tv',
+    'NICK': 'USERNAME',
+    'PASSWORD': 'PASSWORD',
```
Comment (bernardopires) [inline settings.py.example]: Is the format no longer oauth? Please add a comment saying which format is expected. Thanks!

File: settings.py.example [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
+# If you're using a local database the host is 'localhost'
 DATABASE = {
     'NAME': 'twitch',
     'USER': 'postgres',
     'PASSWORD': 'root',
     'HOST': 'db',
 }
 
+API = {
+	'CLIENTID': YOUR_CLIENT_ID
```
Comment (bernardopires) [inline settings.py.example]: Please add quotes if it's a string and add which format is expected (or where to get it from)

**Review by bernardopires — CHANGES_REQUESTED** [review]
Awesome, thanks for the patch! Please add these little comments and we're good to go. Thanks for fixing this!

File: settings.py.example (hunk 1 above)
Comment (juanpflores) [reply settings.py.example]: You're absolutely right.

**Comment by juanpflores** [comment]
@bernardopires I've finished the changes on the Pull Request

**Comment by juanpflores** [comment]
Hey @bernardopires!
Any updates on this PR?

**Review by bernardopires — APPROVED** [review]

**Comment by bernardopires** [comment]
Oops, sorry, forgot to re-review your code. Thanks for the patch!

**Pull request merged**